packages/frontend/src/app/api/content.md

```diff
@@ -183,6 +192,32 @@ GET /validators
         appVersion: 1,
         blockVersion: 13
         validator: "F60A6BF9EC0794BB0CFD1E0F2217933F4B33EDE6FE810692BC275CA18148AEF0"
+      },
+      proTxInfo: {
+        "type": "Evo",
```

Comment:
Keep field names without "", like in other places

Reply:
Done